src/views/SignupSuccess.vue

```diff
@@ -1,170 +1,93 @@
 <template>
     <div class="container pad-bottom">
         <SmallPage>
-            <PageHeader :progressIndicator="true" :numberSteps="numberSteps" :step="numberSteps">
-                Your wallet is ready
-            </PageHeader>
-            <PageBody>
-                <div class="success-box nq-icon trumpet nq-green-bg">
-                    <h2 class="nq-h2">Awesome!</h2>
-                    <p class="nq-text">Your Keyguard Wallet is set up. It already contains your newly created account.</p>
-                    <p class="nq-text">You can add more accounts to it later.</p>
-                </div>
-
-                <div class="wallet-label">
-                    <div class="wallet-icon nq-icon" :class="walletIconClass"></div>
-                    <Input :value="walletLabel" @changed="onWalletLabelChange"/>
-                </div>
-
-                <Account :address="createdAddress.toUserFriendlyAddress()" :label="accountLabel" :editable="true"
-                         @changed="onAccountLabelChange" v-if="!!createdAddress"/>
-
-                <button class="nq-button green submit" @click="done()">Open your wallet</button>
-            </PageBody>
+            <Loader :title="title" :state="state" :lightBlue="true">
+                <template slot="success">
+                    <div class="success nq-icon"></div>
+                    <h1 class="title nq-h1">Welcome to the<br>Nimiq Blockchain.</h1>
+                </template>
+            </Loader>
         </SmallPage>
     </div>
 </template>
 
 <script lang="ts">
 import { Component, Prop, Vue } from 'vue-property-decorator';
-import { PageHeader, PageBody, Account, SmallPage } from '@nimiq/vue-components';
+import { SmallPage } from '@nimiq/vue-components';
 import { AccountInfo } from '../lib/AccountInfo';
 import { WalletInfo, WalletType } from '../lib/WalletInfo';
 import { State } from 'vuex-class';
 import { WalletStore } from '@/lib/WalletStore';
-import { CreateResult } from '@nimiq/keyguard-client';
-import { SignupResult } from '@/lib/RequestTypes';
-import Input from '@/components/Input.vue';
-import { WALLET_DEFAULT_LABEL_KEYGUARD, WALLET_DEFAULT_LABEL_LEDGER, ACCOUNT_DEFAULT_LABEL_KEYGUARD,
-    ACCOUNT_DEFAULT_LABEL_LEDGER } from '@/lib/Constants';
-
-@Component({components: {PageHeader, PageBody, Account, Input, SmallPage}})
+import { OnboardingResult } from '@/lib/RequestTypes';
+import Loader from '@/components/Loader.vue';
+import {
+    WALLET_DEFAULT_LABEL_KEYGUARD,
+    WALLET_DEFAULT_LABEL_LEDGER,
+    ACCOUNT_DEFAULT_LABEL_KEYGUARD,
+    ACCOUNT_DEFAULT_LABEL_LEDGER,
+} from '@/lib/Constants';
+
+@Component({components: {SmallPage}})
 export default class SignupSuccess extends Vue {
-    private static readonly STEPS_KEYGUARD_SIGNUP = 6;
-    private static readonly STEPS_LEDGER_SIGNUP = 3;
-
     @Prop({ default: null })
-    public createResult!: CreateResult;
```

Comment:
actually I would prefer to remove ambient types completely

Reply:
PS: TO make importing less cumbersome, we could export/import KeyguardRequest (equals same as before) instead of all the requests and results for themselves.

Reply:
Removing ambient types should be it's own PR.